testbed/testbed/child_process_collector.go

```diff
@@ -172,7 +172,11 @@ func (cp *childProcessCollector) Start(params StartParams) error {
 	exePath := expandExeFileName(cp.AgentExePath)
 	exePath, err := filepath.Abs(exePath)
 	if err != nil {
-		return err
+		panic(fmt.Sprintf("Failed to resolve the absolute path of expected otel binary. %s", err))
+	}
+
+	if _, err = os.Stat(exePath); err != nil {
+		panic(fmt.Sprintf("The expected otel binary %s is not yet created. %s", exePath, err))
```

Comment:
Hmmm, Looking at this, it should be fine to return an error here instead.

The error should correctly bubble up and I don't want to encourage an anti pattern in the project.